packages/react-inbox/src/components/Inbox/index.tsx

```diff
@@ -297,13 +297,16 @@ const Inbox: React.FunctionComponent<InboxProps> = (props) => {
 
 const InboxWrapper: React.FunctionComponent<InboxProps> = (props) => {
   const courierContext = useCourier<{ inbox: InboxProps }>();
+  const contextExists = Boolean(courierContext);
 
-  if (!courierContext) {
-    console.warn("Inbox: Missing Courier Provider");
-    return null;
-  }
+  return useMemo(() => {
```

Comment:
@rileylnapier Does this need to be memo'd?

Reply:
creates infinite renders without because 'courierContext' changes a lot